.github/workflows/release.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Packages
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        required: true
+        description: Version of the artifact to publish
+      prerelease:
+        type: boolean
+        default: false
+        required: false
+        description: Pre-release
+    secrets:
+      AZURE_VSC_EXT_TOKEN:
+        required: false
+        description: Token to publish Visual Studio Code extension
+      NPM_TOKEN:
+        required: false
+        description: PAT to publish on npmjs.org
+      AZURE_AUTH_TOKEN:
+        required: false
+        description: Token of the user publishing on Azure Feeds for PR Artifacts
+      AZURE_USERNAME:
+        required: false
+        description: Name of the user publishing on Azure Feeds for PR Artifacts
+      AZURE_EMAIL:
+        required: false
+        description: Email of the user publishing on Azure Feeds for PR Artifacts
+  release:
+    types: [published]
+
+jobs:
+  publish-packages:
+    environment: 'production'
+    permissions:
+      packages: write
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup
+        with:
+          enable-build-cache: 'false'
+          install-jdk: 'true'
+      - run: yarn build:swagger-gen
+      - run: yarn build
+      - run: yarn set:version ${{ inputs.version }}
+      - name: Get tag name
+        id: get-npm-tag
+        uses: ./.github/actions/get-npm-tag
+        with:
+          is-prerelease: ${{ inputs.prerelease || (github.event.release && github.event.release.prerelease) }}
```

Comment:
or `is-prerelease: ${{ github.event.release && github.event.release.prerelease }}`

Reply:
done